amr-wind/turbulence/LES/Kosovic.cpp

```diff
@@ -18,7 +18,7 @@ Kosovic<Transport>::Kosovic(CFDSim& sim)
     , m_vel(sim.repo().get_field("velocity"))
     , m_rho(sim.repo().get_field("density"))
     , m_Nij(sim.repo().declare_field("Nij", 9, 1, 1))
-    , m_divNij(sim.repo().declare_field("divNij", 3))
+    , m_divNij(sim.repo().declare_field("divNij", 3, 1, 1))
```

Comment:
I thought @marchdf changed this line as a bugfix in #1030, is there a reason to change this back?

Reply:
I updated to latest repo updated and included the changes. Not sure why it changed back.

Reply:
@hgopalan -- can this conversation be resolved or are things still outstanding?

Reply:
Resolved.